### PR TITLE
add  juniper graphql integration examples

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by fafhrd91** [comment]
Awesome! Thanks

**Comment by ghost** [comment]
Also this issue is being tracked here.
Will need to complete 1 more example on getting to work in sync mode properly.
https://github.com/graphql-rust/juniper/issues/149

**Comment by fafhrd91** [comment]
my understanding from graphql-rust/juniper#149 is sync, should we use sync actor for iteration then? otherwise any juniper request would block whole thread.

**Comment by theduke** [comment]
I think this would actually be a great candidate for an official juniper_ integration crate.

A la `juniper_actix`.
Would be happy to include it in the main juniper repo.

**Comment by fafhrd91** [comment]
I think this is good idea, but it should wait. Example depends on unreleased version of actix-web, I’ll try to release new version during this week

**Comment by ghost** [comment]
Yes.. Thats how it should. This was just an initial poc to figure out how it works with actix-web. I also did the sync actor approach but ran into some issues. Will fix them and push later this week.

File: examples/juniper-websocket/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[package]
+name = "juniper-websocket"
+version = "0.1.0"
+authors = ["Nikolay Kim <fafhrd91@gmail.com>"]
```
Comment (fafhrd91) [inline examples/juniper-websocket/Cargo.toml:4]: you should put your name

**Review by fafhrd91 — APPROVED** [review]

**Comment by fafhrd91** [comment]
are you planing to make more changes? or should I just merge it?

**Comment by fafhrd91** [comment]
could you rebase against master. I made some changes to websockets api

**Comment by ghost** [comment]
Done. Made sure that schema belonged to the actor. You can merge now.

**Comment by fafhrd91** [comment]
something is wrong with branch. it merged a lot of my commits

**Comment by ghost** [comment]
Yeah. I was wondering the same thing.

**Comment by fafhrd91** [comment]
could you maybe create new fork from master and cherry pick your commits into it

**Comment by ghost** [comment]
Yeah will.

**Comment by fafhrd91** [comment]
thanks

**Pull request closed without merging**